src/vivarium_gates_nutrition_optimization/components/pregnancy.py

```diff
@@ -1,26 +1,48 @@
+import pandas as pd
+from vivarium.framework.population import SimulantData
 from vivarium_public_health.disease import DiseaseModel, DiseaseState, SusceptibleState
 
-from vivarium_gates_nutrition_optimization.constants import data_keys, models
+from vivarium_gates_nutrition_optimization.constants import models
 
 
 class NotPregnantState(SusceptibleState):
     def __init__(self, cause, *args, **kwargs):
         super(SusceptibleState, self).__init__(cause, *args, name_prefix="not_", **kwargs)
 
 
+class PregnantState(DiseaseState):
+    def get_initial_event_times(self, pop_data: SimulantData) -> pd.DataFrame:
+        return pd.DataFrame(
+            {self.event_time_column: self.clock(), self.event_count_column: 1},
```

Comment:
Should the event time column be clock time + step size? @rmudambi ?

Reply:
I've talked with Patrick and he will test which is correct

Reply:
you're right, we only get 39 weeks of pregnant time with this.